src/Dropdown/test/DropdownItemSpec.tsx

```diff
@@ -87,7 +87,18 @@ describe('<Dropdown.Item>', () => {
       </Dropdown>
     );
 
-    expect(screen.getByTestId('item')).to.contain(screen.getByTestId('icon'));
+    expect(screen.getByTestId('icon')).to.contain(screen.getByTestId('icon'));
```

Comment:
icon contains icon is logically incorrect, it is recommended to use getByRole('treeitem) instead.

Reply:
Done, there's no need to make any modifications to this change.

Reply:
`Unable to find an accessible element with the role "menuitem"` error is caused by the DropdownItem not being visible, you need to set an open attribute for the Dropdown.
```js
it('Should render a icon without overriding className', () => {
    render(
      <Dropdown open>
        <DropdownItem icon={<div className="custom-class" />} />
      </Dropdown>
    );
    expect(screen.getByRole('menuitem')).to.contain('.custom-class');
  });
```

According to the testing-library's suggestion, we prefer to use `getByRole` to get the element.
https://testing-library.com/docs/queries/about#priority

Reply:
done